src/server/blankett/components/TidligereHistorikk.tsx

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
-import type { IGrunnlagsdataPeriodeHistorikk } from '../../../typer/dokumentApiBlankett';
-import {
-  periodetypeTilTekst,
-  type ITidligereVedtaksperioder,
-  EPeriodetype,
+import type {
+  IGrunnlagsdataPeriodeHistorikkBarnetilsyn,
+  IGrunnlagsdataPeriodeHistorikkOvergangsstønad,
+  ITidligereVedtaksperioder,
 } from '../../../typer/dokumentApiBlankett';
+import { EPeriodetype, periodetypeTilTekst } from '../../../typer/dokumentApiBlankett';
 import { formaterIsoDato, mapBooleanTilString } from '../../utils/util';
 
 export const TidligereHistorikk: React.FC<{
   tidligereVedtaksperioder: ITidligereVedtaksperioder | undefined;
 }> = ({ tidligereVedtaksperioder }) => {
   const periodeHistorikkOvergangsstønad =
     tidligereVedtaksperioder?.sak?.periodeHistorikkOvergangsstønad;
-
+  const periodeHistorikkBarnetilsyn = tidligereVedtaksperioder?.sak?.periodeHistorikkBarnetilsyn;
   const TidligereHistorikk: React.FC = () => {
     return (
       <>
         <p>Har bruker tidligere vedtaksperioder i EF Sak eller Infotrygd?</p>
-        <h3 className={'blankett'}>Overgangsstønad</h3>
+        <h3>Overgangsstønad</h3>
```

Comment:
className={'blankett'} fjernet? Noen grunn til det?

Reply:
La disse tilbake nå

Reply:
Ble lagt til i brev for å skille på css for baks og ef-blankett

Reply:
Åja, du mener at vi skal fjerne className ?